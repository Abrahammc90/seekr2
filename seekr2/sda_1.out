GROUP = Type_Calculation
    type = sda_2proteins
    total_solutes = 2
    total_grids = 2
END GROUP

dseed = 0
nrun =  100000
timemax = 0
probep = 1.4
probew = 1.2
hexcl = 0.1
threshold = 0.0
epfct = 0.5
edfct = 0.36
hdfct = -0.013
save_access = 1

GROUP = ReactionCriteria
    computation = association
    rxna12f = p2.rxna
    dind = 0
    nnnons = 1
    nwrec = 1
    sdamd = 1
END GROUP

GROUP = RateCalculation
    win0 = 3.0
    nwin = 35
    dwin = 0.5
    nb_contact = 1
    bootstrap = 1
    fpt = 0
    stop_traj = 1
    analytical_correction = 0
END GROUP

GROUP = Solute_Grid
    nb_solute = 1
    pdb_filename = /home/abraham/test_mmvt_openmm_sda/b_surface/hostguest_receptor_noh.pdb
    trans_diffusion = 0.03555
    rot_diffusion = 0.000573
    rotate = 1
    surface = 0
    epf = /home/abraham/test_mmvt_openmm_sda/b_surface/hostguest_receptor_ep.grd
    qef = /home/abraham/test_mmvt_openmm_sda/b_surface/hostguest_receptor.echa
    edf = /home/abraham/test_mmvt_openmm_sda/b_surface/hostguest_receptor_ed.grd
    hdf = /home/abraham/test_mmvt_openmm_sda/b_surface/hostguest_receptor_hd.grd
END GROUP

GROUP = Solute_Grid
    nb_solute = 1
    pdb_filename = /home/abraham/test_mmvt_openmm_sda/b_surface/hostguest_ligand_noh.pdb
    trans_diffusion = 0.10859999999999999
    rot_diffusion = 0.01591
    rotate = 1
    surface = 0
    epf = /home/abraham/test_mmvt_openmm_sda/b_surface/hostguest_ligand_ep.grd
    qef = /home/abraham/test_mmvt_openmm_sda/b_surface/hostguest_ligand.echa
    edf = /home/abraham/test_mmvt_openmm_sda/b_surface/hostguest_ligand_ed.grd
    hdf = /home/abraham/test_mmvt_openmm_sda/b_surface/hostguest_ligand_hd.grd
END GROUP

GROUP = Geometry
    type = sphere
    pbc = 0
    surface = 0
    escape = 1
    start_pos = 100
    c = 200
END GROUP

GROUP = Timestep
    variable = 1
    dt1 = 1
    swd1 = 60
    dt2 = 20
    swd2 = 100
END GROUP

GROUP = Complexes
    fcomplexes = complexes_1
    restart_complex = 0
    binary_complex = 0
    nb_complexes = 500
    rmsd_min = 1.0
    ftrajectories = traj_1
    binary_trajectory = 0
    ntraj_rec = -1
    freq_print = 1000
END GROUP

rboost = 1.0

novers = 150

