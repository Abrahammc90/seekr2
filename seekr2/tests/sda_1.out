GROUP = Type_Calculation
    type = sda_2proteins
    total_solutes = 2
    total_grids = 2
END GROUP

dseed = 0
nrun =  100
timemax = 0
probep = 1.4
probew = 1.2
hexcl = 0.1
threshold = 0.0
epfct = 0.5
edfct = 0.36
hdfct = -0.013
save_access = 1

GROUP = ReactionCriteria
    computation = association
    rxna12f = p2.rxna
    dind = 0
    nnnons = 1
    nwrec = 1
    sdamd = 1
END GROUP

GROUP = RateCalculation
    win0 = 3.0
    nwin = 35
    dwin = 0.5
    nb_contact = 1
    bootstrap = 1
    fpt = 0
    stop_traj = 1
    analytical_correction = 0
END GROUP

GROUP = Solute_Grid
    nb_solute = 1
    pdb_filename = /tmp/pytest-of-abraham/pytest-12/hostguest_mmvt1/b_surface/hostguest_receptor_noh.pdb
    trans_diffusion = 0.027630000000000002
    rot_diffusion = 0.0002751
    rotate = 1
    surface = 0
    epf = /tmp/pytest-of-abraham/pytest-12/hostguest_mmvt1/b_surface/hostguest_receptor_ep.grd
    qef = /tmp/pytest-of-abraham/pytest-12/hostguest_mmvt1/b_surface/hostguest_receptor.echa
    edf = /tmp/pytest-of-abraham/pytest-12/hostguest_mmvt1/b_surface/hostguest_receptor_ed.grd
    hdf = /tmp/pytest-of-abraham/pytest-12/hostguest_mmvt1/b_surface/hostguest_receptor_hd.grd
END GROUP

GROUP = Solute_Grid
    nb_solute = 1
    pdb_filename = /tmp/pytest-of-abraham/pytest-12/hostguest_mmvt1/b_surface/hostguest_ligand_noh.pdb
    trans_diffusion = 0.05994
    rot_diffusion = 0.002754
    rotate = 1
    surface = 0
    epf = /tmp/pytest-of-abraham/pytest-12/hostguest_mmvt1/b_surface/hostguest_ligand_ep.grd
    qef = /tmp/pytest-of-abraham/pytest-12/hostguest_mmvt1/b_surface/hostguest_ligand.echa
    edf = /tmp/pytest-of-abraham/pytest-12/hostguest_mmvt1/b_surface/hostguest_ligand_ed.grd
    hdf = /tmp/pytest-of-abraham/pytest-12/hostguest_mmvt1/b_surface/hostguest_ligand_hd.grd
END GROUP

GROUP = Geometry
    type = sphere
    pbc = 0
    surface = 0
    escape = 1
    start_pos = 100
    c = 200
END GROUP

GROUP = Timestep
    variable = 1
    dt1 = 1
    swd1 = 60
    dt2 = 20
    swd2 = 100
END GROUP

GROUP = Complexes
    fcomplexes = complexes_1
    restart_complex = 0
    binary_complex = 0
    nb_complexes = 500
    rmsd_min = 1.0
    ftrajectories = traj_1
    binary_trajectory = 0
    ntraj_rec = -1
    freq_print = 1000
END GROUP

rboost = 1.0

novers = 150

